app/src/ui/app.tsx

```diff
@@ -419,7 +419,6 @@ export class App extends React.Component<IAppProps, IAppState> {
     }
 
     const tip = state.state.branchesState.tip
-    const existsOnRemote = state.state.aheadBehind !== null
```

Comment:
Hmm, it looks like this branch could use a merge in from master?

Reply:
No I've changed the way the delete branch popup checks if the branch is on the remote in a29e2d4bdf85d7b13f758b1eb9ed94c6da662010.

The check is no performed in a more correct way without requiring the `existsOnRemote` prop being passed around:
https://github.com/desktop/desktop/blob/a29e2d4bdf85d7b13f758b1eb9ed94c6da662010/app/src/ui/delete-branch/delete-branch-dialog.tsx#L64-L67

Reply:
Ah, I see. Could we keep that fix separate from this enhancement?

Reply:
I can create a new PR for a29e2d4bdf85d7b13f758b1eb9ed94c6da662010 if you like? https://github.com/desktop/desktop/compare/master...JordanMussi:fix/3469

Reply:
Yes please!

Reply:
Done! #3600 A nice ⚪️ number there! 😮

Reply:
🤘 Mind removing those changes from this PR?

Reply:
If I remove them building this PR will fail as it is dependant on #3600?